### DISCUSSION TITLE
How to use image input for AiServices

### DISCUSSION THREAD

**Discussion by benzourry** [langchain4j/langchain4j#644] [Q&A]
Hi,
I've seen the example of using image input for GPT_4_VISION_PREVIEW here: https://github.com/langchain4j/langchain4j-examples/blob/main/open-ai-examples/src/main/java/OpenAiChatModelExamples.java

Is it possible to use image input using AiServices?

**Comment by bowen0801** [comment]
I tried using ImageContent to construct a UserMessage and pass it in, but AiServices doesn't seem to recognize it, and the responses from the large model are all wrong。

**Comment by dliubarskyi** [comment]
Please check https://docs.langchain4j.dev/tutorials/ai-services#multimodality
